TypeScriptBetaTest/SnippetCompileBetaTests.cs

```diff
@@ -81,7 +81,7 @@ public static void TestsSetUp()
         public static IEnumerable<TestCaseData> TestDataV1 => TestDataGenerator.GetTestCaseData(
```

Comment:
variable name still refers to V1.
```suggestion
        public static IEnumerable<TestCaseData> TestDataBeta => TestDataGenerator.GetTestCaseData(
```